File: STRIVER/video/vid-5/pattern13/p13.py
#  1
#  2 3 
#  4 5 6
#  7 8 9 10


def p13(row:int)->None:
    iter = 1
    for i in range(row+1):
        for j in range(i):
            print(iter,end=" ")
            iter+=1
        print("")

row = 4
p13(row)

File: STRIVER/video/vid-5 (Patterns)/pattern13/p13.cpp
// 1
// 2 3 
// 4 5 6
// 7 8 9 10

#include <iostream>
using namespace std;

int main()
{
    int row = 4;

    int iter = 1;
    for (int i = 1; i <= row; i++)
    { 
        for (int j = 1; j <= i; j++)
        {
            cout << iter << " ";
            iter++;
        }
        cout<<endl;
    }

    return 0;
}